paper_demo/xpcs_client.py

```diff
@@ -91,8 +87,8 @@ def arg_parse():
             'execution_metadata_file': execution_metadata_file,
 
             # funcX endpoints
-            "funcx_endpoint_non_compute": "0ad11dc6-db91-42ce-ab1d-032cdc414582",
-            "funcx_endpoint_compute": "1666324d-163f-4f1f-a374-5038824f9810",
+            "funcx_endpoint_non_compute": "",
+            "funcx_endpoint_compute": "",
```

Comment:
Same as above, commenting these will raise an error in Gladier, instead of allowing the flow to start with empty values.

Reply:
I didn't comment them? I just removed the unknown endpoint uuids. I assume they were Rafs? It is unlikely they will be usable?

Reply:
Sorry, I meant make those lines comments, as in: 

```
# Fill in your funcx endpoints below:
# "funcx_endpoint_non_compute": "",
# "funcx_endpoint_compute": "",
```

If they're blank values, Gladier will start the flow. If they're not present, Gladier will raise an exception saying "Hey, you need to include `funcx_endpoint_compute`". It's a nice catch if people try to run the script without reading the instructions.

Reply:
Whoops, missed your earlier comment. All good then.